cilium/cmd/helpers.go

```diff
@@ -97,6 +98,16 @@ func requirePath(cmd *cobra.Command, args []string) {
 	}
 }
 
+func requireConfigName(cmd *cobra.Command, args []string) {
```

Comment:
This function should be in `endpoint_config_get.go` since it appears to only be used there.